src/PluginForm/Bluesnap/HostedPaymentFieldsPaymentMethodAddForm.php

```diff
@@ -28,6 +29,13 @@ class HostedPaymentFieldsPaymentMethodAddForm extends BasePaymentMethodAddForm {
    */
   protected $clientFactory;
 
+  /**
+   * The fraud session service.
+   *
+   * @var Drupal\commerce_bluesnap\FraudSessionInterface
```

Comment:
**CHANGE REQUEST**

var needs to start with `/`.

Reply:
Corrected